packages/enzyme-adapter-react-renderer/src/ReactTestRendererAdapter.js

```diff
@@ -0,0 +1,46 @@
+import { EnzymeAdapter } from 'enzyme';
+import { createMountWrapper } from 'enzyme-adapter-utils';
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+class ReactTestRendererAdapter extends EnzymeAdapter {
```

Comment:
This is pretty much a streamlined version of `ReactSixteenAdapter`. Because `enzyme` never interacts with this class, I haven't tried very hard to encapsulate ReactDOM operations in here but that could be possible.